### DISCUSSION TITLE
PDF documents in Lychee

### DISCUSSION THREAD

**Discussion by elito** [LycheeOrg/Lychee#2337] [Q&A] [answered]
Is it possible to add .pdf docs in the list of enabled format ?

**Comment by ildyria** [comment]
If you add it to your `raw_formats` list ([`|` separated if I remember correctly](https://github.com/LycheeOrg/Lychee/blob/e534bc83a45927fdf31f79d431c7050886d48318/app/Image/Files/BaseMediaFile.php#L292)), you should be able to upload them. That being said, I don't think we support making previews out of them.
Also pdf is not really a _photo format_. :/

**Reply by elito** [reply]
OK i know but in few cases this could be useful to uploads some documents. No preview is not a problem in my case.

Thank for your reply.

**Reply by elito** [reply]
So after testing for information pdf thumbs have preview but no preview in zoom

**Comment by ildyria** [comment, marked as answer]
done
